plio/io/io_spectral_profiler.py

```diff
@@ -82,12 +82,31 @@ def __init__(self, input_data, cleaned=True, qa_threshold=2000):
                               count=nrows)
             self.ancillary_data = pd.DataFrame(d, columns=columns,
                                                index=np.arange(nrows))
-
+            keys = []
+            vals = []
+            for k, v in label.items():
+                if k in ["ANCILLARY_AND_SUPPLEMENT_DATA", "L2D_RESULT_ARRAY",
+                         "SP_SPECTRUM_QA", "SP_SPECTRUM_REF1", "SP_SPECTRUM_RAD",
+                         "SP_SPECTRUM_REF2", "SP_SPECTRUM_RAW", "SP_SPECTRUM_WAV",
+                         "^ANCILLARY_AND_SUPPLEMENT_DATA", "^SP_SPECTRUM_WAV",
+                         "^SP_SPECTRUM_RAW", "^SP_SPECTRUM_REF2"," ^SP_SPECTRUM_RAD",
+                         "^SP_SPECTRUM_REF1", "^SP_SPECTRUM_QA", "^L2D_RESULT_ARRAY",
+                         "^SP_SPECTRUM_RAD"]:
+                    continue
```

Comment:
petty detail probably, but for readability, would it make sense to stuff the list into a variable before the if statement?